tests/struct.c

```diff
@@ -1,23 +1,54 @@
 #include <stdio.h>
 #include <stdlib.h>
-
+int len;
```

Comment:
What is your motivation to modify `tests/struct.c`?

Reply:
Some of the struct operate will failed, so I improve the struct test case

Reply:
Send another pull request to concentrate on struct test case changes.